### ISSUE TITLE
Abnormal synaptic physiology 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7054]
**Preferred term label:**

Abnormal synaptic physiology 

**Synonyms**

**Definition (free text, please give PubMed ID)**

Any functional anomaly of the membrane junction site of a nerve cell to a target cell, such as another nerve cell, an effector cell, or a sensory receptor cell; transmission of nerve impulses may be mediated by chemical or by electrical means.
PMID:30439458

**Parent term (use hpo.jax.org/app)**
abnormal nervous system physiology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by nicolevasilevsky** [comment]
Thanks @sarahstofel. This new term request looks good.

Tiny minor things:
- could you please write the PMIM without a space, ie PMID:30439458
- could you please write the term labels with a capital first letter, ie, Abnormal synpatic physiology

thank you, we appreciate your contributions.

**Comment by pnrobinson** [comment]
Please see https://github.com/obophenotype/human-phenotype-ontology/issues/7055, I think this is out of scope, but please reopen with more details if appropriate.
